htsget.md

```diff
@@ -162,6 +162,28 @@ The server SHOULD reply with an `UnsupportedFormat` error if the requested forma
 [^a]
 </td></tr>
 <tr markdown="block"><td>
+
+`class`  
+_optional string_
+</td><td>
+
+Request different classes of data.
+By default, i.e., when `class` is not specified, the response will represent a complete read or variant data stream, encompassing SAM/CRAM/VCF headers, body data records, and EOF marker.
+
+If `class` is specified, its value MUST be one of the following:
+<table>
+<tr><td>
+
+`header`
+</td><td>
+
+Request the SAM/CRAM/VCF headers only.
```

Comment:
@jmarshall What do you think of adding "(and EOF marker)" here? Useful clarification, thanks for bringing it up earlier.